service/workflow.py

```diff
@@ -74,6 +89,9 @@ def process_issn(cls, account_id, issn):
                     # if the above worked, then we can update the harvest state
                     if saved:
                         state.set_harvested(p.get_name(), lhd)
+        except (CaughtTermException, KeyboardInterrupt):
+            app.logger.warning(u"Harvester caught SIGTERM while processing ISSN:{x} for Account:{y} ".format(y=account_id, x=issn))
```

Comment:
This is good for now, but you might want to refactor what is being logged into a function. This is for outputting a summary when the process ends (whether by SIGTERM or reaching its normal end of life). https://github.com/DOAJ/harvester/issues/3#summaryofwhathappenedduringrun